test/test_nn.py

```diff
@@ -9690,6 +9698,16 @@ def test_upsamplingBiLinear2d_consistency(self, device, memory_format, antialias
             input_ui8, size=(output_size, output_size), mode=mode, align_corners=align_corners, antialias=antialias
         )
 
+        # Check if output is CF for unsqueezed 3d CL tensor
+        if check_as_unsqueezed_3d_tensor and memory_format == torch.channels_last:
+            self.assertTrue(input_ui8.is_contiguous(memory_format=torch.channels_last))
+            self.assertTrue(output_ui8.is_contiguous())
+            self.assertTrue(output_f32.is_contiguous())
```

Comment:
These two lines are worth a comment indicating that while this is the behaviour that we currently have, it is **definitely unexpected** and ideally this would be fixed such that both the ui8 and f32 outputs are also channels_last. Might even be worth prefixing with a `FIXME`